src/components/structured-text-block/structured-text-block.vue

```diff
@@ -148,6 +148,21 @@
           },
         })
       }
+      case 'StructuredTextImageRecord': {
+        return h(ImageWithCaption, {
+          class: {
+            'structured-text__image-with-caption': true,
+            'structured-text__image-with-caption--narrow': record.layout === 'narrow',
```

Comment:
Since narrow causes the image to be a bit smaller, what do you think of using a different `sizes` property too?

Reply:
Ah you're right. I did think about that, but forgot to do so in the end 🙈